seed/challenges/01-responsive-web-design/claim-your-responsive-web-design-certificate.json

```diff
@@ -8,9 +8,9 @@
       "title": "Claim Your Responsive Web Design Certificate",
       "description": [
         [
-          "//i.imgur.com/k8btNUB.jpg",
+          "//i.imgur.com/GjTPLxI.jpg",
```

Comment:
This looks like it uses the old format with more than one signature - can you update it? i.imgur.com/GjTPLxI.jpg

Reply:
Yeah, I have that in the issue description, waiting until everything else is good (design etc.) to prevent repeating the same process again and again 👍